<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depozit</class>
 <widget class="QWidget" name="depozit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Сумма вклада</string>
   </property>
   <widget class="QLineEdit" name="contribution">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5000000</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>₽</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>$</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>€</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Срок размещения</string>
   </property>
   <widget class="QLineEdit" name="time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Месяц</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Год</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Процентная ставка</string>
   </property>
   <widget class="QLineEdit" name="procent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Налоговая ставка</string>
   </property>
   <widget class="QLineEdit" name="tax">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Периодичность выплат</string>
   </property>
   <widget class="QComboBox" name="payment_per">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ежедневно</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ежемесячно</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Капитализация процента</string>
   </property>
   <widget class="QComboBox" name="payment_flag">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Нет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ежедневно</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ежемесячно</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Дата открытия</string>
   </property>
   <widget class="QLineEdit" name="date_open">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2000:01:01</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>16</width>
     <height>501</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Пополнение:</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сколько?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="add">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>₽</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>$</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> €</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Когда?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_time">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0000:00:00</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Снятие:</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сколько?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sub">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Когда?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sub_time">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0000:00:00</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>₽</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>$</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> €</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="add_add">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_sub">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="run">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_10">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Сумма на вкладе в конце</string>
   </property>
   <widget class="QTextEdit" name="sum_end">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_11">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Начисленные проценты</string>
   </property>
   <widget class="QTextEdit" name="sum_procent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_12">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>260</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Сумма налога</string>
   </property>
   <widget class="QTextEdit" name="sum_tax">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="add_list">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sub_list">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>130</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
